components/bookingForm/bookingForm.module.css

```diff
@@ -0,0 +1,7 @@
+.alert {
+  @apply relative mb-4 rounded border-[1px] border-solid border-transparent p-4;
+}
+
+.alert-success {
+  @apply border-[#badbcc] bg-[#d1e7dd] text-[#0f5132];
```

Comment:
are there any "red"s that we can reference instead of hardcoding the colors here?